src/Magick.NET.Core/Types/Percentage.cs

```diff
@@ -18,14 +18,22 @@ namespace ImageMagick;
     /// </summary>
     /// <param name="value">The value (0% = 0.0, 100% = 100.0).</param>
     public Percentage(double value)
-        => _value = value;
+    {
+        Throw.IfNegative(nameof(value), value);
```

Comment:
Maybe I should have used `Throw.IfOutOfRange` with [0.0,100.0] range?

Reply:
I may have found a counter argument to do it:

https://github.com/dlemstra/Magick.NET/blob/483e8753f16f5b2401c9eb447e5e9e091fdb8749/tests/Magick.NET.Tests/MagickImageTests/TheResizeMethod.cs#L94-L105

Reply:
Same for `Scale` method.

Reply:
Good that I have some unit tests for that :smile:. Percentages above 100% are allowed.

Reply:
At least description of Percentage should be updated 😅

Reply:
Maybe we should change it to `(0% = 0.0, 100% = 100.0, 142.42% = 142.42)`?